asciidoc/modules/con_mg_changes-in-json.adoc

```diff
@@ -118,7 +137,39 @@ Use the following example code to convert a Base64URL to Base64 encoder.
 [source,java,options="nowrap",subs="attributes+"]
 ----
 String base64url = someJsonObject.getString("base64encodedElement")
-String base64 = Json.toBase64(base64url);
+String base64 = toBase64(base64url);
+----
+
+The helper functions described above address plain JSON migrations. If you use automatic object mapping from JSON
+to a {Java} POJO, then a custom object mapper needs to be created to your specific use case.
+
+An object mapper with a custom Base64 decoder:
```

Comment:
The following example shows you how to create a object mapper with custom Base64 decoder.